test/param-name-reference.xspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description query="http://example.org/ns/my" query-at="subtract.xqm" stylesheet="subtract.xsl"
+	xmlns:my="http://example.org/ns/my" xmlns:x="http://www.jenitennison.com/xslt/xspec">
+
+	<x:scenario
+		label="Two instances of function-param (i.e. //x:call[@function]/x:param), the latter referencing the former,">
```

Comment:
Interesting. I don't think it ever occurred to me to refer to a non-global `x:param` in a following sibling `x:param`.

Reply:
I never thought about this use case until I tried fixing this problem :)